src/Razor/benchmarks/Microsoft.AspNetCore.Razor.Microbenchmarks/LanguageServer/RazorLanguageServerBenchmarkBase.cs

```diff
@@ -28,13 +29,19 @@ public RazorLanguageServerBenchmarkBase()
         Logger = new NoopLogger();
         RazorLanguageServer = RazorLanguageServerWrapper.Create(serverStream, serverStream, Logger, NoOpTelemetryReporter.Instance, configure: (collection) =>
         {
-            collection.AddSingleton<ClientNotifierServiceBase, NoopClientNotifierService>();
```

Comment:
Doesn't this break the two existing breakpoints that override `Builder`, since they're not calling the base method?